fairmq/shmem/Monitor.cxx

```diff
@@ -309,17 +314,14 @@ void Monitor::CheckSegment()
                  << setw(10) << (fViewOnly ? "view only" : to_string(duration)) << " |"
                  << c << flush;
         } else if (fViewOnly) {
-            time_t current = chrono::system_clock::to_time_t(now);
-            chrono::milliseconds ms = chrono::duration_cast<chrono::milliseconds>(now.time_since_epoch());
-            struct tm local;
-            localtime_r(&current, &local);
-            char timeBuffer[80];
-            size_t count = strftime(timeBuffer, 80, "%F %T", &local);
-            cout << (count != 0 ? timeBuffer : "") << "." << setfill('0') << setw(6) << ms.count() % 1000000
-                 << ", name: " << fSegmentName
-                 << ", size: " << segment.get_size()
-                 << ", free: " << segment.get_free_memory()
-                 << ", numDevices: " << numDevices << endl;
+            size_t free = segment.get_free_memory();
+            size_t total = segment.get_size();
+            size_t used = total - free;
+            LOGV(info, user1) << "[" << fSegmentName
```

Comment:
Can you please double-check that the min required FairLogger version in the top-level CMakeLists.txt is high enough for LOGV. If not, bump it pls.

Reply:
Nice catch. Need at least 1.6. I'll bump it.
Looks like it was broken already. In tests we also use `Logger::DefineVerbosity` and `VerbositySpec` that were introduced in FairLogger 1.4, but our requirement is 1.2.